src/main/app/controller/AppRequest.ts

```diff
@@ -24,6 +24,7 @@ export interface AppSession extends Session {
   lang: string | undefined;
   errors: FormError[] | undefined;
   addresses: [];
+  fee: string;
```

Comment:
can we store whole fee object instead of just the amount
that will help with less modifications in common files later

Reply:
Updated as per review comments.